e2e/framework/contracts/interface_docker_client.go

```diff
@@ -0,0 +1,8 @@
+package contracts
+
+type IDockerClient interface {
```

Comment:
the interfaces shouldn't have the `I` prefix or be in a filename with `interface`. we can open an issue after the PR though